src/main/java/io/vertx/ext/mongo/GridFsUploadOptions.java

```diff
@@ -3,15 +3,20 @@
 import io.vertx.codegen.annotations.DataObject;
 import io.vertx.core.json.JsonObject;
 
+import java.util.Objects;
+
 @DataObject
 public class GridFsUploadOptions {
+  private static final Integer DEFAULT_CHUNK_SIZE_IN_BYTES = 255 * 1000;
```

Comment:
Where does this new default come from? How was the default computed before?

Reply:
GridFS uses a default chunk size of 255kB, see https://docs.mongodb.com/manual/core/gridfs/

Before it was simply `null`.

I am getting confused tbh, what is the way to go with that changes. Should we integrate defaults defined by the driver into this project, or simply pass null on that occasions? It's apparently not done consistently throughout the project.

Reply:
We try to be consistent as much as possible but it's not fundamental. Perhaps sometimes the original author had good reasons to let the driver pick its own defaults.

Reply:
I changed it back to `null` to be on the safe side then.